examples/scaling-criteo/01-Download-Convert.ipynb

```diff
@@ -69,22 +69,24 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 3,
    "metadata": {},
    "outputs": [
     {
-     "data": {
-      "application/vnd.jupyter.widget-view+json": {
-       "model_id": "933e7da7339647308a9b3cd0ca4a6b3a",
-       "version_major": 2,
-       "version_minor": 0
-      },
-      "text/plain": [
-       "downloading day_1.gz: 0.00B [00:00, ?B/s]"
-      ]
-     },
-     "metadata": {},
-     "output_type": "display_data"
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "downloading day_0.gz: 16.3GB [30:56, 8.79MB/s]                                  \n",
```

Comment:
Do we want to change this output @benfred ?

Reply:
@benfred what is your opinion? I am not even sure what the related issue could be?